testgrid/config.yaml

```diff
@@ -5629,6 +5632,12 @@ dashboards:
     test_group_name: ppc64le-unit
     description: unit test results for ppc64le
 
+- name: sig-node-arm64
+  dashboard_tab:
+  - name: conformance
+    test_group_name: arm64-conformance
+    description: conformance test results for arm64
```

Comment:
can you provide a bit more detail for this? what kubernetes version? using what tools? see EG https://github.com/kubernetes/test-infra/pull/9315/files https://github.com/kubernetes/test-infra/pull/8965/files

Reply:
I have changed it. Please check.